extensions/wikia/PortableInfobox/PortableInfoboxHooks.class.php

```diff
@@ -1,8 +1,9 @@
 <?php
+
 class PortableInfoboxHooks {
 	// FIX ME: temporary implementation - styles should be included only on page with portable infobox
```

Comment:
Not related to this commit, but should be changed to TODO: Add to global css on sitewide release